XamlControlsGallery/SamplePages/SamplePage7.xaml.cs

```diff
@@ -0,0 +1,43 @@
+using Windows.Foundation.Metadata;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Media.Animation;
+using Windows.UI.Xaml.Navigation;
+
+namespace AppUIBasics.SamplePages
+{
+    public sealed partial class SamplePage7 : Page
+    {
+        public SamplePage7()
+        {
+            this.InitializeComponent();
+        }
+
+        public void PrepareConnectedAnimation(ConnectedAnimationConfiguration config)
```

Comment:
Does this page need the ConnectedAnimation stuff? If not, please remove. My guess is that it's copy-pasta from another sample page? If it's not needed, please remove.